extensions/dacpac/package.json

```diff
@@ -57,7 +57,9 @@
 	},
 	"devDependencies": {
 		"@types/mocha": "^5.2.5",
-		"mocha": "^5.2.0",
+    "mocha": "^5.2.0",
```

Comment:
spacing looks off here

Reply:
Hmm doesn't look off to me, might be a tabs vs spaces kinda thing.

Reply:
I always enable space visualization for this reason 😄 

![image](https://user-images.githubusercontent.com/28519865/65794206-7f255e00-e11c-11e9-974f-69c7916b61e2.png)